dev/NavigationView/NavigationView.h

```diff
@@ -74,38 +81,56 @@ class NavigationView :
     int GetNavigationViewItemCountInTopNav();
     winrt::SplitView GetSplitView();
     TopNavigationViewDataProvider& GetTopDataProvider() { return m_topDataProvider; };
-    winrt::ListView LeftNavListView() { return m_leftNavListView.get(); };
     void TopNavigationViewItemContentChanged();
 
     void CoerceToGreaterThanZero(double& value);
 
+    void RaiseItemInvoked(winrt::IInspectable const& item,
+        bool isSettings,
+        winrt::NavigationViewItemBase const& container = nullptr,
+        NavigationRecommendedTransitionDirection recommendedDirection = NavigationRecommendedTransitionDirection::Default);
+    void RaiseItemInvokedForNavigationViewItem(const winrt::NavigationViewItem& nvi);
+    void OnNavigationViewItemInvoked(const winrt::NavigationViewItem& nvi);
+
+    bool IsSettingsItem(winrt::IInspectable const& item);
+
+    static winrt::DependencyProperty GetNavigationViewItemRevokersProperty()
+    {
+        static GlobalDependencyProperty s_NavigationViewItemRevokersProperty =
+            InitializeDependencyProperty(
```

Comment:
>InitializeDependencyProperty [](start = 12, length = 28)

What's the reason it's not generated by code gen?

Reply:
This is an implementation detail, so I didn't want to define it in the IDL file (and hence have it be generated by codegen through there)

---
In reply to: [353481602](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r353481602) [](ancestors = 353481602)